packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -202,6 +202,12 @@ class HeapTreeViewState extends State<HeapTree>
 
   void _initTabs() {
     _tabs = [
+      if (enableNewAllocationProfileTable)
+        DevToolsTab.create(
+          key: dartHeapTableTabKey,
+          tabName: 'Profile',
```

Comment:
thoughts on renaming to 'Allocation Profile'? @polina-c @InMatrix

Reply:
I believe tab names should be short for easiness of reference. We have context 'Memory'. So it is Memory Profile, that is the same as Memory Allocation Profile. How about taking this discussion out of this PR?